### PR TITLE
Add responsive email format to the babylon-notifier

### PR DESCRIPTION
**Pull request by makirill**
MJML is a framework to generate responsive emails (more details at https://mjml.io/)

There are two major changes needed to introduce MJML framework to the `babylon-notifier`:
1. mjml cli needs to be installed, so it can be used by the `operator.py` script
2. Jinja2 templates should generate valid mjml format that will be used to generate final HTML for the email notifications.

I did the "success" path testing (provisioning start -> environment ready -> environment deleted). Deployment was done with `odo` tool.

### CODE REVIEW

File: notifier/operator/templates/base.mjml.j2 [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <mj-section padding-top="0" padding-bottom="0" padding-left="0" padding-right="0">
+      <mj-column>
+        <mj-spacer height="25px" />
+      </mj-column>
+    </mj-section>
+    <!-- Title -->
+    <mj-section padding-top="21px" padding-bottom="12px" padding-left="12px" padding-right="12px">
+      <mj-column>
+        <mj-text font-family="Red Hat Text" font-weight="regular" align="center" font-size="16px" color="#000" line-height="1.2">
+          Status: Service Ready
```
Comment (aleixhub) [inline notifier/operator/templates/base.mjml.j2:21]: should this be better placed under the content block?

Comment (makirill) [reply notifier/operator/templates/base.mjml.j2]: Good catch, I've fixed it with 75d1a3bb7e54a6de9db003c978327d6543755d58

File: notifier/operator/templates/base.mjml.j2 [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <mj-section padding-top="0" padding-bottom="0" padding-left="0" padding-right="0">
+      <mj-column>
+        <mj-spacer height="25px" />
+      </mj-column>
+    </mj-section>
+    <!-- Text -->
+    <mj-section padding-top="12px" padding-bottom="12px" padding-left="12px" padding-right="12px" background-color="#f0f0f0">
+      <mj-column>
+        <mj-text font-family="Red Hat Text" color="#4a5557" font-size="16px" line-height="1.5" padding-bottom="12px">
+          {% block content %}{% endblock %}
```
Comment (aleixhub) [inline notifier/operator/templates/base.mjml.j2:42]: suggestion: Lift the content to be inside the mj-section
And then in every .mjml template use the mj-column / mj-text accordingly. As it is now the templates will not have lots of flexibility as they are encapsulated under those elements and they use plain html syntax without having the benefits of mjml syntax

Comment (makirill) [reply notifier/operator/templates/base.mjml.j2]: Good suggestion. I wanted to implement it this way initially but for some reason I've change my mind and left `child` templates with pure HTML/JINJA2 without any MJML tags.

New commit should fix it - 164ccd9d6571e7ce4eea603084fd785f60f9eb7b

**Review by aleixhub — APPROVED** [review]
LGTM, great work

**Comment by aleixhub** [comment]
merged in https://github.com/redhat-cop/babylon/pull/2187

**Pull request closed without merging**